intellij-pants-plugin.iml

```diff
@@ -1,106 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file can be removed, as Gradle perfectly replaces DevKit

Reply:
And in the follow-up PR we could save "gradle:runIde" runConfiguration in the git

https://www.jetbrains.com/help/idea/run-debug-configuration.html#save-as-permanent

Reply:
Removed